src/pykeen/nn/init.py

```diff
@@ -481,8 +481,19 @@ def __init__(
         super().__init__(tensor=tensor)
 
 
-#: A resolver for initializers, including both elements of :mod:`torch.nn.init` and
-#: custom additions in :mod:`pykeen.nn.init`
+# TODO: replace by automatically generated list
```

Comment:
class-resolver probably needs a function that can be run to update the docstring of a variable after it's been made. I don't think we can hack something into FunctionResolver that is able to update the docstring of the variable it's assigned into

Reply:
I think the easiest way would be to "just" have the optional to generate some rst table or similar and add a reference to this table here.